linetools/spectra/lsf.py

```diff
@@ -330,8 +333,24 @@ def load_STIS_data(self):
             data_aux = ascii.read(file_name, data_start=2, names=col_names)
             # reformat rel_pix
             data_aux['rel_pix'] = self.check_and_reformat_relpix(data_aux['rel_pix'])
+
+            # handle asymmetric STIS LSF
+            if data_aux['rel_pix'][len(data_aux['rel_pix']) // 2] == 0.:
+                pass
+            elif data_aux['rel_pix'][(len(data_aux['rel_pix']) // 2) - 1 ] ==0.:
+                data_aux.insert_row(0,data_aux[-1])
+                data_aux['rel_pix'][0]=-data_aux['rel_pix'][-1]
+            elif data_aux['rel_pix'][(len(data_aux['rel_pix']) // 2) + 1 ] ==0.:
+                data_aux.add_row(data_aux[0])
+                data_aux['rel_pix'][-1]=-data_aux['rel_pix'][0]
+
             # create column with absolute wavelength based on pixel scales
```

Comment:
I suppose this is a philosophical question: I'm more inclined to leave the data tables from STScI intact.  In the spirit of reproducibility, if someone were to download linetools and, with it, these tables from STScI with the exact file names and *nearly* the same data, I see how I would be really confused if trying to understand how the code works and where the data come from.